tests/Task/YamlConfigurationTaskTest.php

```diff
@@ -276,4 +277,16 @@ public function testOnInstallReleaseCreateOrUpdateConfigurationFailsCreatingANew
         $task = new YamlConfigurationTask('/parameters.yml', array('foo' => 'bar', 'baz' => '', 'bar' => array('baz' => '')), array('baz', 'bar.baz'));
         $task->onInstallReleaseCreateOrUpdateConfiguration($event, AccompliEvents::INSTALL_RELEASE, $eventDispatcherMock);
     }
+
+    /**
+     * Test setting a value generator.
+     */
+    public function testValueGeneratorSetter()
+    {
+        $task = new YamlConfigurationTask('/parameters.yml');
+        $generator = new SecretGenerator();
```

Comment:
Changing this to a `ValueGeneratorInterface` mock will not give the `SecretGenerator` false coverage.